Contributing.md

```diff
@@ -82,4 +82,32 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
+
+The Express TC can designate captains for individual projects/repos in the
+three organizations. These captains are responsible for being the primary
+day-to-day maintainers of the repo on a technical and community front.
+They will ensure that decisions which effect the Express project at large are
+brought up with the TC (in meeting or over GitHub issues) but otherwise will
+have the full authority in the direction of the individual project.  These captains
+will be given full permissions on both the repo as well as the npm packages, and
+will follow the community guidelines for publishing and maintaining support.
+
+The Express TC can designate captains for individual projects/repos in the
```

Comment:
@gireeshpunathil these two paragraphs are super similar.  I think the intent is to only have one of them.

Reply:
@ryhinchey - thanks for pointing out, removed the redundant one (copy past error!), PTAL!